dev/CommonStyles/TestUI/MenuFlyoutPage.xaml

```diff
@@ -10,36 +10,62 @@
     mc:Ignorable="d">
 
     <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" Margin="12">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto"/>
+            <ColumnDefinition Width="*"/>
+        </Grid.ColumnDefinitions>
+        <StackPanel Grid.Column="1" HorizontalAlignment="Center">
+            <StackPanel Orientation="Horizontal">
+                <TextBlock Text="MenuFlyoutItem height: "/>
+                <TextBlock x:Name="TestMenuFlyoutItemHeightTextBlock" AutomationProperties.Name="TestMenuFlyoutItemHeightTextBlock" Text="0" Margin="5, 0, 0, 0"/>
+            </StackPanel>
+            <StackPanel Orientation="Horizontal" Margin="0,20,0,0">
+                <TextBlock Text="MenuFlyoutItem width: "/>
+                <TextBlock x:Name="TestMenuFlyoutItemWidthTextBlock" AutomationProperties.Name="TestMenuFlyoutItemWidthTextBlock" Text="0" Margin="5, 0, 0, 0"/>
+            </StackPanel>
+        </StackPanel>
+
         <StackPanel>
```

Comment:
Add Columnspan=2 to this stackpanel if you're adding columns to the parent grid.